src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -43,6 +43,7 @@ public static partial class PlatformDetection
         public static bool IsRedHatFamily7 => false;
         public static bool IsNotRedHatFamily6 => true;
 
+        // Windows 10 Anniversary release a.k.a Windows 10 Version 1607 added support to native WinHTTP for HTTP/2 protocol support.
```

Comment:
Why is this comment here?

Reply:
I could see this being useful information, but it probably belongs with the actual tests that are being disabled.

Reply:
To make note for WinHttpHandler HTTP/2 testing. 

For example, Windows 8.1 supports ALPN, but the native WinHTTP doesn't have HTTP/2 support. Thus SocketsHttpHandler can communicate using HTTP/2 on Windows 8.1, but WinHttpHandler cannot due to native WinHTTP limitation.

Reply:
> but it probably belongs with the actual tests that are being disabled.

The comment could save people's time digging around to find proper Windows version in PlatformDetection class.

Reply:
I think this comment is useful but this file is not the right place for the comment.  The comment could go somewhere in a file that has WinHttpHandler HTTP/2 tests in it where it is using the `IsWindowsVersion10Version1607OrGreater` property to skip/not-skip the test.

Reply:
Will delete the comment here, and move it to #31440 (which has WinHttpHandler HTTP/2 tests).